public/dashboard.css

```diff
@@ -38,123 +48,149 @@ div.stat-box {
+}
+
 div.puzzles {
-  flex: 1;
-  display: flex;
-  flex-direction: column;
-  background-color: #c0c0ff;
-  overflow: hidden;
+  grid-row: 1 / 7;
+  grid-column: 2;
+  border: 16px solid #1976d2;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(12, 1fr);
+  grid-gap: var(--dashboard-radius);
 }
 
 div.puzzle {
-  width: 100%;
+  font-size: 40px;
+  display: grid;
+  grid-gap: calc(var(--dashboard-radius) * 2);
+  grid-template-columns: 64px 2fr min-content;
+  width: calc(100% - (4 * var(--dashboard-radius)));
+  margin: 0 auto;
+  padding: var(--dashboard-radius);
+  justify-self: center;
+  align-self: center;
+  align-items: center;
+  border: 2px solid #e1e1e1;
+}
+
+div.puzzle:has(span.pin) {
+  background-color: #ffecb3;
+  color: #880e4f;
+  font-weight: bold;
 }
 
 div.puzzle-status-emoji {
-  float: left;
-  width: 4vw;
-  max-width: 4vw;
-  font-size: 5vh;
-  text-align: center;
+  display: block;
+  aspect-ratio: 1 / 1;
+  font-size: 56px;
 }
 
-div.puzzle-title-and-topic {
-  float: left;
-  width: 25vw;
-  max-width: 25vw;
-  display: flex;
-  flex-direction: column;
+div.puzzle-details {
   white-space: nowrap;
   overflow: hidden;
+  text-overflow: ellipsis;
 }
 
-span.puzzle-title {
-  font-size: 3vh;
-  white-space: nowrap;
-  overflow: hidden;
+div.puzzle-header{
+  display: flex;
+  align-items: center;
 }
 
-span.puzzle-topic {
-  font-size: 2vh;
+span.puzzle-title {
   white-space: nowrap;
   overflow: hidden;
+  text-overflow: ellipsis;
 }
 
-div.puzzle-tags {
-  float: left;
-  width: 10vw;
-  max-width: 10vw;
-  display: flex;
-  flex-wrap: wrap;
-  white-space: nowrap;
-  overflow: hidden;
+span.puzzle-tags {
+  font-size: 0.5em;
+  margin-left: 1rem;
 }
 
 span.tag {
-  flex: 0;
-  height: 2.5vh;
-  line-height: 2.5vh;
-  padding: 0.1vh 0.2vw;
-  margin: 0.2vh 0.1vw;
-  border-radius: 0.5vh;
-  text-decoration: none;
-  background-color: #004d40;
-  color: #ffffff;
-  font-size: 1.5vh;
+  font-weight: normal;
   white-space: nowrap;
+  padding: 4px 8px;
+  margin-right: 4px;
+  background-color: #004d40;
+  color: white;
+  border-radius: var(--dashboard-radius);
 }
 
 span.tag[class*="meta/"] {
   background-color: #0d47a1;
 }
+
 span.tag[class*="in/"] {
   background-color: #616161;
 }
+
 span.tag[class*="priority/"] {
   background-color: #880e4f;
 }
 
-div.puzzle-people {
+span.puzzle-topic {
+  font-size: 0.55em;
+  font-weight: normal;
   float: left;
-  width: 10vw;
-  max-width: 10vw;
-  max-height: 6.5vh;
-  display: flex;
-  flex-wrap: wrap;
+  max-width: 100%;
   white-space: nowrap;
   overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+div.puzzle-people {
+  max-width: max-content;
+  justify-self: right;
+  direction: rtl;
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  grid-template-columns: 1fr;
+  grid-auto-columns: 1fr;
+  grid-auto-flow: column;
+  overflow: hidden;
 }
 
 div.puzzle-people img {
```

Comment:
I do like the profile pics, but we should figure out some way to cap the number of these we show (and maybe do like a +N thing for the number we dropped, although that'll probably be more complicated)